config/secrets.yml

```diff
@@ -19,9 +19,15 @@
 
 development:
   secret_key_base: 3a114eb06c65ebbf342f77cbce85917a821387f80603dd9883bb92aadd160bc4c09c1494ecf7489d138f34ae0422f690fa2ee47b41e89bb4badeb3a9088d4954
+  twilio_account_sid:  <%= ENV.fetch("SECRET_KEY_BASE", "AC7f2af091f4a2bbed0a98517827216672") %>
+  twilio_auth_token:  <%= ENV.fetch("SECRET_KEY_BASE", "2dce6bb8d9bbd9b98a991b2daf26028f") %>
+  twilio_phone_number:  <%= ENV.fetch("SECRET_KEY_BASE", "14155238973") %>
 
```

Comment:
This file should be ignored.

Reply:
Done. @peanutenthusiast

Reply:
I can see the .gitignore in files changed..